third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -412,7 +412,23 @@ function Target_topFrameLocation() {
  */
 function getStackFrames() {
   // NOTE: this is a custom command we added in `src/inspector/v8-debugger-agent-impl.cc`
-  const { callFrames } = sendMessage("Debugger.getCallFrames");
+  const value = sendMessage("Debugger.getCallFrames");
+  if (value === undefined) {
+    return [{
```

Comment:
I think this should just throw (from inside sendMessage), since we'll catch that at https://github.com/replayio/chromium/blob/master/third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc#L285 and return an empty object.

Reply:
That would also allow us to handle this more generally, since right now this patch means that every single `sendMessage` callsite would need to be taught how to handle an `undefined` result (or else throw a random JS error, like this code would have)